FlowCrypt/Functionality/Services/AppStartup.swift

```diff
@@ -152,7 +152,18 @@ struct AppStartup {
         ) { _ in
             self.initializeApp(window: window)
         }
+        let logout = UIAlertAction(
+            title: "log_out".localized,
+            style: .default
+        ) { _ in
+            do {
+                try appContext.globalRouter.signOut(appContext: appContext)
+            } catch let logoutError {
+                Logger.logError("Logout failed due to \(logoutError.localizedDescription)")
```

Comment:
Better here would be to make another alert - let the user know what happened. It doesn't have to have any buttons anymore. Alternatively, we have a "fatal error" VC that you could send the user with the error message.

Because from the perspective of the user, the logout button would appear to do nothing.

However, it's an unlikely situation within an already unlikely situation, so can leave it as is.

Reply:
Got it